pages/common/tldr.md

```diff
@@ -14,3 +14,7 @@
 - Get help for a git subcommand:
 
 `tldr {{git-checkout}}`
+
+- Update the local tldr pages:
```

Comment:
```suggestion
- Update local pages (if the client supports caching):
```
Simplify, use "pages" instead of "tldr pages". Also mention that client needs to support caching in order for the `-u` option to be available.